apps/src/dropletUtils.js

```diff
@@ -275,9 +277,9 @@ exports.generateDropletPalette = function (codeFunctions, dropletConfig) {
     var blockPair = {
       block: block,
       expansion: expansion,
-      title: cf.func
+      title: funcInfo.func
     };
-    mergedCategories[cf.category].blocks.push(blockPair);
+    mergedCategories[funcInfo.category].blocks.push(blockPair);
```

Comment:
This block of changes is not strictly necessary, and should be a no-op, but I don't like seeing variables like "cf"